yb-voyager/src/query/sqltransformer/sqlfiletransformer.go

```diff
@@ -161,9 +161,12 @@ func (t *IndexFileTransformer) writeRemovedRedundantIndexesToFile(removedIndexTo
 // TODO: merge the sharding/colocated recommendation changes with this Table file transformation
 type TableFileTransformer struct {
 	//skipping the merge constraints with this parameter
-	skipMergeConstraints         bool
-	sourceDBType                 string
-	skipPerformanceOptimizations bool
+	skipMergeConstraints           bool
+	sourceDBType                   string
+	skipPerformanceOptimizations   bool
+	PKConstraintsOnTimestampOrDate []string
+	OtherPKConstraints             []string
+	AppliedShardingChanges         bool
```

Comment:
this field is sort of ambiguously named. It could also mean sharding/colocated
How about `AppliedShardingTypeConfigurations` ? or `AppliedHashRangeConfigurations` ?